tune_sklearn/_trainable.py

```diff
@@ -82,17 +82,20 @@ def _setup_early_stopping(self):
         self.fold_train_scores = np.empty(n_splits, dtype=dict)
         if self.early_stop_type == EarlyStopping.WARM_START_ITER:
             # max_iter here is different than the max_iters the user sets.
-            # max_iter is to make sklearn only fit for one epoch,
-            # while max_iters (which the user can set) is the usual max
-            # number of calls to _trainable.
+            # max_iter is to make sklearn only fit for max_iter/max_iters
+            # epochs, while max_iters (which the user can set) is the usual
+            # max number of calls to _trainable.
             self.estimator_config["warm_start"] = True
-            self.estimator_config["max_iter"] = 1
+            self.estimator_config[
+                "max_iter"] = self.main_estimator.max_iter // self.max_iters
```

Comment:
do you need to cast to integer?

Reply:
Floor division of an integer by integer returns an integer